notification-hubs-test-app/src/main/java/com/microsoft/notification_hubs_test_app/NotificationData.java

```diff
@@ -10,9 +10,8 @@
 import android.graphics.Paint;
 import android.media.RingtoneManager;
 import android.net.Uri;
-import android.os.Bundle;
-import android.support.v4.app.NotificationCompat;
-import android.widget.ImageView;
+
```

Comment:
Why the whitespace here?

Reply:
Different sections for `android` and `androidx`. Auto-generated change by migration tool, but I agree. There's no need.

Reply:
Fixed in e8e996ed3dfbcb66acc8b60ebdacdb737eec9279